star-rx72n-firmware/lib/rx_bus/src/rx_bus_config.c

```
@@ -15,36 +15,45 @@
 
 #include <string.h>
 
-#include "hardware_pinout.h"
 #include "rx_check.h"
 #include "rx_log.h"
+#include "rx_port_constants.h"
 
 static const char* s_tag = "BUS_CFG";
 
+static rx_err_t
+internal_validate_port_pin(const uint8_t port, const uint8_t pin_num, const char* context_tag)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_validate_port_pin' of similar type ('const int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)